app/controllers/presentations_controller.rb

```diff
@@ -9,7 +9,12 @@ def new
   end
 
   def create
-    @presentation = Presentation.new(presentation_params)
+    # because of the presentation policy we know a person
+    # will have to be signed in to view this page.
+    # current_person will never be nil
```

Comment:
Which page is `this page`?
Do you think the reader would expect that `current_user` _could_ be `nil`?
What is expected to happen if I POST to `/presentations` without logging in? Can this be tested for in the controller spec?